### PR TITLE
add speed potion to autopot

### PR DESCRIPTION
**Pull request by yoseph1247**

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleAutoPot.kt [hunk 1]
```diff
@@ -164,13 +167,19 @@ object ModuleAutoPot : Module("AutoPot", Category.COMBAT) {
...
+
         if (regenPotion && healthIsLow && !player.hasStatusEffect(StatusEffects.REGENERATION)) {
             allowedStatusEffects += StatusEffects.REGENERATION
         }
+
         if (strengthPotion && !player.hasStatusEffect(StatusEffects.STRENGTH)) {
             allowedStatusEffects += StatusEffects.STRENGTH
         }
 
+        if (speedPotion) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleAutoPot.kt]: Might check if the player not already has the speed effect.

**Review by 1zun4 — CHANGES_REQUESTED** [review]
The code simply does not compile.

**Comment by Ell1ott** [comment]
What about doing it like this: https://github.com/Ell1ott/LiquidBounce/commit/7c2c3527697b0f874fdf7e58c015a361730ec862

**Comment by yoseph1247** [comment]
👍

**Review by Ell1ott — APPROVED** [review]

**Review by 1zun4 — APPROVED** [review]

**Comment by yoseph1247** [comment]
i'm not 100% sure on how to fix this compiling issue, i'm not the greatest at coding

**Comment by be4dev** [comment]
> i'm not 100% sure on how to fix this compiling issue, i'm not the greatest at coding

do you use IDEA?

**Comment by yoseph1247** [comment]
> > i'm not 100% sure on how to fix this compiling issue, i'm not the greatest at coding
> 
> do you use IDEA?

yeah but i wasn't using it for this because it's very laggy, seems to be fixed now and it does compile so

**Comment by be4dev** [comment]
> > > i'm not 100% sure on how to fix this compiling issue, i'm not the greatest at coding
> > 
> > 
> > do you use IDEA?
> 
> yeah but i wasn't using it for this because it's very laggy, seems to be fixed now and it does compile so

you can try vscode or nvim with kotlin support plugin

**Comment by yoseph1247** [comment]
> > > > i'm not 100% sure on how to fix this compiling issue, i'm not the greatest at coding
> > > 
> > > 
> > > do you use IDEA?
> > 
> > 
> > yeah but i wasn't using it for this because it's very laggy, seems to be fixed now and it does compile so
> 
> you can try vscode or nvim with kotlin support plugin

it worked well enough, just gotta give it some extra time

**Comment by yoseph1247** [comment]
guess it doesnt compile but it compiles in my own ide...

**Comment by ghost** [comment]
It's a detekt issue, it doesn't like how the code is presented in the isPotion function. Don't worry, I will try to fix it rq...

**Comment by ghost** [comment]
👍

**Pull request merged**